bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/AbstractTreeViewer.java

```diff
@@ -3401,4 +3396,38 @@ public boolean contains(Object parent, Object element) {
 		return false;
 	}
 
+	/**
+	 * Transactionally perform an action on the TreeViewer without redrawing
+	 * regularly. Disables redraw, executes toExecute and enables redraw again.
```

Comment:
nit: put `toExecute` in `@code{toExecute}` to make it easier to see that it's a code reference